fre/frepp/configUpdates/pp.yaml

```diff
@@ -0,0 +1,225 @@
+## User defined edits
+configuration_paths: 
+  rose-suite: "/home/Dana.Singh/pp/ppyaml/fre-cli/fre/frepp/configUpdates/rose-suite-am5_c96L33_amip.conf" 
+  rose-remap: "/home/Dana.Singh/pp/ppyaml/fre-cli/fre/frepp/configUpdates/remap-rose-app.conf"
+  rose-regrid: "/home/Dana.Singh/pp/ppyaml/fre-cli/fre/frepp/configUpdates/regrid-rose-app.conf"
+
+analysisLen: &ANA_AMIP_LEN "10yr"
+chunk96: &PP_AMIP_CHUNK96 "1yr"
+analysisStart: &ANA_AMIP_START "1980"
+analysisStop: &ANA_AMIP_STOP "1988"
+defxyInterp: &defaultxyInterp "180,288"
```

Comment:
Neat... these are like FRE properties, user variables that can be reused later?

I see the value set there, e.g. `10yr`, but is the variable `analysisLen` or `ANA_AMIP_LEN`?

Reply:
Yes! They're used later in the components section. It was slightly confusing, the variable is defined as `analysisLen` in the yaml, but the value actually reused later, that saves the value the user sets, is the one with the `&`